cmake/Linux.cmake

```diff
@@ -30,6 +30,10 @@ set(PNET_SNMP_PRIO 1
   CACHE STRING "SNMP thread priority")
 set(PNET_SNMP_STACK_SIZE 256*1024
   CACHE STRING "SNMP thread stack size")
+set(PNET_SNMP_OBJECT_ID NETSNMP_SYSTEM_MIB
+  CACHE STRING "Value of SNMP sysObjectID")
+set(PNET_SNMP_SYS_SERVICES 78
+  CACHE STRING "Value of SNMP sysServices")
```

Comment:
I was under the impression that it always should be 78 for Profinet devices. If not include some example and a reference.
"Value of SNMP sysServices. Typically 78 for profinet devices, but can be for example 71 if ??? is disabled. See ???"

Reply:
All hardcoded constants were moved here. sysServices is not really an option though so should maybe be hardcoded in the file. Are we sure there is never any reason for changing it?

I understood sysObjectID to be vendor-specific, sort of a product ID:

```
 The vendor's authoritative identification of the
network management subsystem contained in the entity.
This value is allocated within the SMI enterprises
subtree (1.3.6.1.4.1) and provides an easy and
unambiguous means for determining `what kind of box' is
being managed. For example, if vendor `Flintstones,
Inc.' was assigned the subtree 1.3.6.1.4.1.424242,
it could assign the identifier 1.3.6.1.4.1.424242.1.1
to its `Fred Router'. 
```

Reply:
I think @fredrikdanielmoller knows more about these details.

Reply:
PN-AL-Protocol ch. 4.16.5 specifies use of 24686 in sysObjectID.

Reply:
Regarding sysServises, the PN-Topology guideline says 

> "The value for PROFINET devices is 78, i.e. 1001110 in binary format"

I couldn't find this in PN-AL-Protocol.

Reply:
Ok, fair enough. Removing both sysObjectID and sysServices.